week6.client/src/components/addstore/AddStorePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+export class AddStorePage extends Component {
+
+    async addStores() {
+
+        let sName = document.getElementById('storeName').value;
+
+        console.log(sName);
```

Comment:
We can use console.log locally for debugging. But we need to remove them in the production level.